R/SubsetQueryBuilders.R

```diff
@@ -49,23 +49,27 @@ CohortSubsetQb <- R6::R6Class(
   inherit = QueryBuilder,
   private = list(
     innerQuery = function(targetTable) {
+      cohortWindowLogic <- lapply(private$operator$windows, function(window) {
+        lsql <- " AND (S.@s_cohort_anchor >= DATEADD(d, @window_start_day, T.@window_anchor) AND S.@s_cohort_anchor <= DATEADD(d, @window_end_day, T.@window_anchor))"
+        SqlRender::render(lsql,
+                          window_anchor = ifelse(window$targetAnchor == "cohortStart",
```

Comment:
If you're inclined, in the CI package the choice of start/end is simply 'start' or 'end' seen here:https://github.com/OHDSI/CohortIncidence/blob/09cf6aa3a6f4cfdbe99d04e38b61a85441fa3925/R/Classes.R#L547.  If we want to adopt that moniker for choosing a start or end date to those choices instead of 'cohortStart' or 'cohortEnd'.  I origionally had CI using same thing (with cohort in the label) but switched it out to be not cohort-specific but just simply a choice of start/end.  Just pointing out where we might be consistent across packages.